configure.ac

```diff
@@ -229,6 +229,10 @@ AS_IF([test "x$with_fs" != "xno" -o "x$with_crypto" != "xno" -o "x$with_swap" !=
              [Define as neutral value if libblkid doesn't provide the definition])])]
       [])
 
+AS_IF([test "x$with_btrfs" != "xno"],
+      [LIBBLOCKDEV_PKG_CHECK_MODULES([LIBBTRFSUTIL], [libbtrfsutil >= 6.6])],
+      [])
```

Comment:
To unblock some of the CI, you'll need to `libbtrfsutil` dependency to few places:
- `dist/libblockdev.spec.in`
- `misc/install-test-dependencies.yml`
- `.packit.yaml`

This will cover Packit and the GH actions checks (we'll also need to add it to our internal CI, but that's something we need to do manually).